ui/src/layout/ActivityPanel.js

```diff
@@ -42,6 +49,31 @@ const useStyles = makeStyles((theme) => ({
   counterStatus: {
     minWidth: '15em',
   },
+  heading: {
+    padding: '0px 16px',
+  },
+  list: {
+    maxWidth: '300px',
+    textOverflow: 'ellipsis',
+    ' & .primary': {
+      '& .MuiListItemText-primary': {
```

Comment:
this means for add following CSS to `.MuiListItemText-primary` wrapped around div with `class = "primary"`